book/src/proposals/simple-payment-and-state-verification.md

```diff
@@ -58,17 +58,17 @@ Clients that want to query the chain for a receipt of the "latest" state would n
 
 Leaders should coalesce the validator votes by stake weight into a single entry. This will reduce the number of entries necessary to create a receipt.
 
-### Chain of Entries
+### Chain of Components in Merkle Path
```

Comment:
`Entries` here is referring to the comprising parts of Merkle Path, not the Entry. Confusing at best, use different word. I picked `component`, no particular preference. `segment` could work too.

Reply:
Here Entries would be referring to the "vector of PoH entries" below. The next few lines are intended to describe how we get from a TX or account state up to the Bank-Merkle, which consequently links to votes.

This whole section could use a rewrite.  It comes from a time before concepts like "rooted bank" and "confidence" were solidified and is admittedly hand wavy as a result.